models/multiphase/d2q9_pf_velocity/Dynamics.R

```diff
@@ -39,9 +39,11 @@ AddField('PhaseF',stencil2d=1, group="PF")
 #	Additional access required for outflow boundaries
 if (Options$Outflow){
 	for (d in rows(DensityAll)){
-    		AddField( name=d$name,  dx=-d$dx-1, dy=-d$dy, dz=-d$dz )
+    		AddField( name=d$name,  dx=-d$dx-1, dy=-d$dy )
+    		AddField( name=d$name,  dx=-d$dx, dy=-d$dy-1 )
```

Comment:
same name?

Reply:
@mdzik This extends the scope avaliable for reading for field.
@TravisMitchell You can also do `AddField( ..., dx=c(-d$dx-1,-d$dx), ...)`